core/deployment/pom.xml

```diff
@@ -73,6 +73,10 @@
             <groupId>org.graalvm.sdk</groupId>
             <artifactId>graal-sdk</artifactId>
         </dependency>
+        <dependency>
```

Comment:
It's needed in order to reference the `JNIRuntimeAccess` class. Can we add this here?